content/applications/hr/time_off.rst

```diff
@@ -281,34 +277,54 @@ Fill out the following fields on the form:
+  each month the milestone occurs.
+- :guilabel:`Cap accrued time`: If there is a maximum amount of days the employee can accrue with
   this plan, enable this option.
 
   When enabled, two additional fields appear beneath it. Select the type of time period from the
   drop-down menu, either :guilabel:`Days` or :guilabel:`Hours`.
 
   Then, enter a numerical value in the field to specify the maximum amount of time that can be
   accrued.
-- :guilabel:`Milestone reached`: enter the number and value of the time period that must pass before
+- :guilabel:`Milestone reached`: Enter the number and value of the time period that must pass before
   the employee starts to accumulate time off. The first value is numerical; enter a number in the
   first field.
 
   Then, select the type of time period using the drop-down menu in the second field. The options
   are: :guilabel:`Days`, :guilabel:`Months`, or :guilabel:`Years`.
 - :guilabel:`Carry over`: select how any unused time off is handled. The options are either:
 
-  - :guilabel:`None. Accrued time reset to 0`: any unused time off is gone.
-  - :guilabel:`All accrued time carried over`: all unused time off is rolled over to the next
+  - :guilabel:`None. Accrued time reset to 0`: Any unused time off is gone.
+  - :guilabel:`All accrued time carried over`: All unused time off is rolled over to the next
     calendar year.
-  - :guilabel:`Carry over with a maximum`: unused time off is rolled over to the next calendar year,
+  - :guilabel:`Carry over with a maximum`: Unused time off is rolled over to the next calendar year,
     but there is a cap. An :guilabel:`Up to` field appears if this is selected. Enter the maximum
     number of :guilabel:`Days` that can roll over to the following year. Any time off beyond this
     parameter is lost.
 
-Once the form is completed, click :guilabel:`Save & Close` to save the form and close the modal, or
-click :guilabel:`Save & New` to save the form and create another milestone. Add as many milestones
-as desired.
+.. important::
+   If the :guilabel:`Carry over` field is set to :guilabel:`None. Accrued time reset to 0`, that
+   rule *overrides* the :guilabel:`Carry-Over Time` set on the accrual plan.
+
+   If a company creates an accrual plan, granting employees time off :guilabel:`At the start of the
+   accrual period` (the beginning of the year), and sets the :guilabel:`Carry-Over Time` on the
+   *accrual plan* to :guilabel:`At the start of the year`, it allows unused vacation time to
+   rollover to the following year.
+
+   Then, the company adds rules to the accrual plan, allocating five days of vacation, annually, on
+   the first of the year (one week of vacation allocated on January 1st).
+
+   If the :guilabel:`Carry over` field is set to :guilabel:`None. Accrual time reset to 0` on the
+   :guilabel:`Create Milestone` pop-up for, any unused vacation time **DOES NOT carry over**, even
```

Comment:
subjective formatting, italics is fine (no bold/caps).
the consequence of what happens here is implied by the label, if we really want to make it a big deal use an admonition block (warning/important)

```suggestion
   :guilabel:`Create Milestone` pop-up for, any unused vacation time *does not* carry over, even
```